app/ui-react/packages/atlasmap-assembly/package.json

```diff
@@ -91,7 +91,7 @@
     "@angular/platform-browser": "7.2.7",
     "@angular/platform-browser-dynamic": "7.2.7",
     "@angular/router": "7.2.7",
-    "@atlasmap/atlasmap-data-mapper": "^1.42.10",
+    "@atlasmap/atlasmap-data-mapper": "^1.43.0",
```

Comment:
I must admit I merged this without looking at it, now I see that we might be missing the `yarn.lock` update for this; we're building with `--frozen-lockfile` IIRC.